frontend/lib/laletterbuilder/faq-content.tsx

```diff
@@ -29,8 +30,8 @@ export const faqContent: FaqItem[] = [
     answer: (
       <span>
         <Trans id="laletterbuilder.faq.whocanhelpv2">
```

Comment:
@shakao you might need to increment/change this id tag here to register that a new translation is needed